### PR TITLE
Add recent command

### PR DESCRIPTION
**Pull request by kanaarima**
implements !recent command.

### CODE REVIEW

**Review by Lekuruu — CHANGES_REQUESTED** [review]
There is an attribute called `preferred_mode`. I would fetch the scores by this mode.
For example:

```python
score = scores.fetch_recent(user_id=user.id, mode=user.preferred_mode, limit=1)
```

File: app/commands/recent.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    n300 = score.n300
+    n100 = score.n100
+    n50 = score.n50
+    nmiss = score.nMiss
+    pp = score.pp
+    nscore = score.total_score
+    mods = Mods(score.mods).short
+
+    embed = Embed(
+        title=f"{beatmapset.title} +{mods}",
```
Comment (Lekuruu) [inline app/commands/recent.py:36]: Should we do `beatmapset.full_name` here, or does that look bad?

Comment (kanaarima) [reply app/commands/recent.py:36]: we should

Comment (Lekuruu) [reply app/commands/recent.py:36]: ok

**Review by Lekuruu — APPROVED** [review]

**Pull request merged**